hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java

```diff
@@ -216,7 +218,17 @@ public static OMClientRequest createClientRequest(OMRequest omRequest,
             omRequest.getDeleteOpenKeysRequest().getBucketLayout());
       }
       return new OMOpenKeysDeleteRequest(omRequest, bktLayout);
-
+    case RecoverLease: {
+      volumeName = omRequest.getRecoverLeaseRequest().getVolumeName();
+      bucketName = omRequest.getRecoverLeaseRequest().getBucketName();
+      bucketLayout =
+              getBucketLayout(ozoneManager.getMetadataManager(), volumeName, bucketName);
+      if (bucketLayout != BucketLayout.FILE_SYSTEM_OPTIMIZED) {
```

Comment:
ozone also have object store, check if included as part of this to cleanup openKeyTable

Reply:
No... i don't think it's a good idea to break the existing object store semantics. No object store support a client lease recovery semantics. This is purely meant for HDFS-like semantics for existing HDFS applications to migrate over to Ozone, so I believe it is exclusively FSO.